ui/src/testing/stubs/institution-api-stub.ts

```diff
@@ -78,10 +78,6 @@ export const VERILY_WITHOUT_CT: Institution = {
       eraRequired: true,
       emailDomains: ['verily.com', 'google.com'],
     },
-    {
```

Comment:
When an Inst does not provide access to a tier, that is returned from the API as a missing tier config, rather than this NOACCESS tier config.